designer/client/components/tips/Errors.js

```diff
@@ -59,7 +59,7 @@ export default class Errors extends React.Component {
 
     //TODO: this is dependent on messages from BE. Should be unified to proper resource bundle :/
     const looseNodeIds = nodeIds.filter(nodeId => nodeErrors[nodeId].some(error => error.message === "Loose node"))
-    const invalidEndNodeIds = nodeIds.filter(nodeId => nodeErrors[nodeId].some(error => error.message === "Invalid end of scenario"))
+    const invalidEndNodeIds = nodeIds.filter(nodeId => nodeErrors[nodeId].some(error => error.message === "Scenario must end with sink or processor"))
```

Comment:
what about the case when scenario ends with fragment ?

Reply:
Message is changed to "end with a sink, processor or fragment".